core-services/image-mirroring/knative/mapping_knative_v0_23_quay

```diff
@@ -11,6 +11,7 @@ registry.ci.openshift.org/openshift/knative-v0.23.0:knative-eventing-test-wathol
 registry.ci.openshift.org/openshift/knative-v0.23.0:knative-eventing-test-wathola-forwarder quay.io/openshift-knative/wathola-forwarder:v0.23
 registry.ci.openshift.org/openshift/knative-v0.23.0:knative-eventing-test-wathola-receiver quay.io/openshift-knative/wathola-receiver:v0.23
 registry.ci.openshift.org/openshift/knative-v0.23.0:knative-eventing-test-wathola-sender quay.io/openshift-knative/wathola-sender:v0.23
+registry.ci.openshift.org/openshift/knative-v0.22.3:knative-eventing-kafka-test-wathola-kafka-sender quay.io/openshift-knative/wathola-kafka-sender:v0.23
```

Comment:
This looks wrong :thinking:. Shouldn't it 0.23.0 in the first image?

Reply:
for Kafka it's actually `0.23.3`, see: https://github.com/openshift/release/blob/master/ci-operator/config/openshift-knative/eventing-kafka/openshift-knative-eventing-kafka-release-v0.23__47.yaml#L180-L184

Reply:
true

Reply:
Fixed.